.github/stale.yml

```diff
@@ -1,5 +1,5 @@
 daysUntilStale: 90
-daysUntilClose: 7
+daysUntilClose: false
```

Comment:
I assume this affects issues and pull requests in the same way? I don't mind either way

Reply:
Yes, it should affect both.